.github/workflows/ci.yaml

```diff
@@ -48,4 +48,4 @@ jobs:
         with:
           submodules: true
       - name: Build and Test
-        run: ./mvnw -q clean verify install
\ No newline at end of file
+        run: ./mvnw -P github -q clean verify deploy
```

Comment:
Don't do this in condition `on.pull_request`, it will always fail in forked branches, and we should only deploy master codes, not PR codes

Reply:
I think we should not deploy in CI.

Reply:
reverted